<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLabel_Class</class>
 <widget class="QDialog" name="AddLabel_Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowMySpeed" stdset="0">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QSpinBox" name="xBox">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="yBox">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="wBox">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="hBox">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
